etc/mtree/README

```diff
@@ -23,8 +23,8 @@ and keep them in good working order.
 
             mkdir /tmp/MTREE
             mtree -deU -f BSD.X.dist -p /tmp/MTREE
-            mtree -cdin -k uname,gname,mode -p /tmp/MTREE | \
-		sed -e 's/ *$//' >BSD.X.new
+            mtree -bcdijn -k uname,gname,mode -p /tmp/MTREE | \
```

Comment:
These instructions were probably designed for fmtree. If so, I'll update the flags accordingly.
`-i` -> `-j`